test/integration/FileBackendEncodedSlashHandling.test.ts

```diff
@@ -0,0 +1,93 @@
+import fetch from 'cross-fetch';
+import type { App } from '../../src/init/App';
+import { getPort } from '../util/Util';
+import {
+  getDefaultVariables,
+  getTestConfigPath,
+  getTestFolder,
+  instantiateFromConfig,
+  removeFolder,
+} from './Config';
+
+const port = getPort('FileBackendEncodedSlashHandling');
+const baseUrl = `http://localhost:${port}/`;
+
+const rootFilePath = getTestFolder('file-backend-encoded-slash-handling');
+
+describe('A server with a file backend storage', (): void => {
```

Comment:
For each test I would add an additional check that looks at the file name on disk since we know what it should be and where it should be located.